packages/server/src/api/controllers/datasource.ts

```diff
@@ -174,14 +173,6 @@ export async function update(
   await events.datasource.updated(datasource)
   datasource._rev = response.rev
 
-  // Drain connection pools when configuration is changed
-  if (datasource.source && !isBudibaseSource) {
-    const source = await getIntegration(datasource.source)
-    if (source && source.pool) {
-      await source.pool.end()
-    }
-  }
-
```

Comment:
As far as I can tell, `source.pool` is never a thing. The only integration that uses a `pool` is MSSQL, and it's an instance property not a static one. Please correct me if I'm wrong @mike12345567 🙏